auctioneer-template/program/src/sell/mod.rs

```diff
@@ -0,0 +1,182 @@
+
+    /// CHECK: Verified through CPI
+    /// Metaplex metadata account decorating SPL mint account.
+    pub metadata: UncheckedAccount<'info>,
+
+    /// CHECK: Verified through CPI
+    /// Auction House authority account.
+    pub authority: UncheckedAccount<'info>,
+
+    /// Auction House instance PDA account.
+    #[account(seeds=[PREFIX.as_bytes(), auction_house.creator.as_ref(), auction_house.treasury_mint.as_ref()], seeds::program=auction_house_program, bump=auction_house.bump, has_one=auction_house_fee_account)]
+    pub auction_house: Box<Account<'info, AuctionHouse>>,
+
+    /// CHECK: Not dangerous. Account seeds checked in constraint.
+    /// Auction House instance fee account.
+    #[account(mut, seeds=[PREFIX.as_bytes(), auction_house.key().as_ref(), FEE_PAYER.as_bytes()], seeds::program=auction_house_program, bump=auction_house.fee_payer_bump)]
+    pub auction_house_fee_account: UncheckedAccount<'info>,
+
+    /// CHECK: Not dangerous. Account seeds checked in constraint.
+    /// Seller trade state PDA account encoding the sell order.
+    #[account(mut, seeds=[PREFIX.as_bytes(), wallet.key().as_ref(), auction_house.key().as_ref(), token_account.key().as_ref(), auction_house.treasury_mint.as_ref(), token_account.mint.as_ref(), &u64::MAX.to_le_bytes(), &token_size.to_le_bytes()], seeds::program=auction_house_program, bump=trade_state_bump)]
+    pub seller_trade_state: UncheckedAccount<'info>,
+
+    /// CHECK: Not dangerous. Account seeds checked in constraint.
+    /// Free seller trade state PDA account encoding a free sell order.
+    #[account(mut, seeds=[PREFIX.as_bytes(), wallet.key().as_ref(), auction_house.key().as_ref(), token_account.key().as_ref(), auction_house.treasury_mint.as_ref(), token_account.mint.as_ref(), &0u64.to_le_bytes(), &token_size.to_le_bytes()], seeds::program=auction_house_program, bump=free_trade_state_bump)]
+    pub free_seller_trade_state: UncheckedAccount<'info>,
+
+    /// CHECK: Verified through CPI
+    /// The auctioneer program PDA running this auction.
+    pub auctioneer_authority: UncheckedAccount<'info>,
+
+    /// CHECK: Not dangerous. Account seeds checked in constraint.
+    /// The auctioneer PDA owned by Auction House storing scopes.
+    #[account(seeds = [AUCTIONEER.as_bytes(), auction_house.key().as_ref(), auctioneer_authority.key().as_ref()], seeds::program=auction_house_program, bump = auction_house.auctioneer_pda_bump)]
+    pub ah_auctioneer_pda: UncheckedAccount<'info>,
+
+    /// CHECK: Not dangerous. Account seeds checked in constraint.
+    #[account(seeds=[PREFIX.as_bytes(), SIGNER.as_bytes()], seeds::program=auction_house_program, bump=program_as_signer_bump)]
+    pub program_as_signer: UncheckedAccount<'info>,
+
+    pub token_program: Program<'info, Token>,
+    pub system_program: Program<'info, System>,
+    pub rent: Sysvar<'info, Rent>,
+}
+
+// /// Create a sell bid by creating a `seller_trade_state` account and approving the program as the token delegate.
```

Comment:
Same thing as above, the only thing that needs to be removed is the listing_config setup and the args related to them. The only purpose of this function will be to call Auction House's auctioneer_sell.